NoteFind/View Controllers/ProfileViewController.m

```diff
@@ -76,15 +77,15 @@ - (void)getOfflineNotes {
 
 - (UICollectionViewCell *)collectionView:(nonnull UICollectionView *)collectionView cellForItemAtIndexPath:(nonnull NSIndexPath *)indexPath {
     if (self.filterSegmentCtrl.selectedSegmentIndex == kOnlineNoteIndex) {
-        OnlineGridCell *cell = [collectionView dequeueReusableCellWithReuseIdentifier:@"OnlineGridCell" forIndexPath:indexPath];
+        OnlineGridCell *cell = [self.collectionView dequeueReusableCellWithReuseIdentifier:@"OnlineGridCell" forIndexPath:indexPath];
```

Comment:
Curious why we have different re-use identifiers if the cell type is the same?

Reply:
there are different re-use identifiers because one uses a PFImageView and the other an imageView, so I set up the pictures differently